cosmwasm/contracts/yield-ward/src/execute/unstake.rs

```diff
@@ -11,16 +11,6 @@ pub fn try_init_unstake(
     token_denom: TokenDenom,
     lpt_amount: Uint128,
 ) -> Result<Response, ContractError> {
```

Comment:
Lacks the `lpt_amount != 0` check

Reply:
Check for this is presented out of the function:
```rust
match action_msg {
    Cw20ActionMsg::Unstake => {
        if msg.amount.is_zero() {
            return Err(ContractError::ZeroTokenAmount);
        }
        let deposit_token_denom =
            find_deposit_token_denom_by_lpt_address(deps.as_ref(), &info.sender)?;
        try_init_unstake(deps, env, user, deposit_token_denom, msg.amount)
    }
}
```